tests/python/unittest/test_operator.py

```diff
@@ -28,6 +28,87 @@
 from common import setup_module, with_seed
 import unittest
 
+def check_rnn_consistency(cell1, cell2, T, N, I, H):
+    dshape = (N, T, I)
+    data = mx.sym.Variable('data')
+
+    Y1, _ = cell1.unroll(T, data, layout='NTC', merge_outputs=True)
+    mod1 = mx.mod.Module(Y1, label_names=None, context=mx.cpu())
```

Comment:
Fixed. Also I changed the name of `test_lstm` to `test_lstm_sym` since it would confict with that in `/unittest/test_gluon_rnn.py` after imported to `test_operator_gpu.py`.